src/coreclr/vm/interoplibinterface.cpp

```diff
@@ -1601,13 +1609,16 @@ bool GlobalComWrappersForTrackerSupport::TryGetOrCreateObjectForComInstance(
         _ASSERTE(hr == S_OK);
     }
 
+    // TrackerObject support and unwrapping matches the built-in semantics that the global scenario mimics.
+    int flags = CreateObjectFlags::CreateObjectFlags_TrackerObject | CreateObjectFlags::CreateObjectFlags_Unwrap;
```

Comment:
It looks like we previously didn't automatically unwrap in the TrackerGlobalInstance scenario but now we will. Is that expected?

Reply:
Nope. That is a mistake. Technically it doesn't matter given how the internal tracker contract works but that is a mistake. Thanks for pointing that out. This means I must have missed something elsewhere...

Reply:
Yay. Didn't miss anything, this was just a mistake. Updated.